pkg/requirements/requirements_test.go

```diff
@@ -7,36 +7,49 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
-
-	"github.com/replicate/cog/pkg/config"
 )
 
-func TestPythonPackages(t *testing.T) {
-	tmpDir := t.TempDir()
-	build := config.Build{
-		PythonPackages: []string{"torch==2.5.1"},
-	}
-	config := config.Config{
-		Build: &build,
-	}
-	_, err := GenerateRequirements(tmpDir, &config)
-	require.ErrorContains(t, err, "python_packages is no longer supported, use python_requirements instead")
-}
-
 func TestPythonRequirements(t *testing.T) {
 	srcDir := t.TempDir()
 	reqFile := path.Join(srcDir, "requirements.txt")
 	err := os.WriteFile(reqFile, []byte("torch==2.5.1"), 0o644)
 	require.NoError(t, err)
 
-	build := config.Build{
-		PythonRequirements: reqFile,
-	}
-	config := config.Config{
-		Build: &build,
-	}
 	tmpDir := t.TempDir()
-	requirementsFile, err := GenerateRequirements(tmpDir, &config)
+	requirementsFile, err := GenerateRequirements(tmpDir, reqFile)
 	require.NoError(t, err)
 	require.Equal(t, filepath.Join(tmpDir, "requirements.txt"), requirementsFile)
 }
+
+func TestReadRequirements(t *testing.T) {
+	srcDir := t.TempDir()
+	reqFile := path.Join(srcDir, "requirements.txt")
+	err := os.WriteFile(reqFile, []byte("torch==2.5.1"), 0o644)
+	require.NoError(t, err)
+
+	requirements, err := ReadRequirements(reqFile)
+	require.NoError(t, err)
+	require.Equal(t, []string{"torch==2.5.1"}, requirements)
+}
+
+func TestReadRequirementsLineContinuations(t *testing.T) {
+	srcDir := t.TempDir()
+	reqFile := path.Join(srcDir, "requirements.txt")
+	err := os.WriteFile(reqFile, []byte("torch==\\\n2.5.1"), 0o644)
+	require.NoError(t, err)
+
+	requirements, err := ReadRequirements(reqFile)
+	require.NoError(t, err)
+	require.Equal(t, []string{"torch==2.5.1"}, requirements)
+}
+
+func TestReadRequirementsStripHashes(t *testing.T) {
```

Comment:
```suggestion
func TestReadRequirementsStripComments(t *testing.T) {
```

Also I think this needs to test line comments not just trailing comments.